substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ConditionalConfigurationRegistry.java

```diff
@@ -38,6 +38,9 @@ public abstract class ConditionalConfigurationRegistry {
     private final Map<String, Collection<Runnable>> pendingReachabilityHandlers = new ConcurrentHashMap<>();
 
     protected void registerConditionalConfiguration(ConfigurationCondition condition, Runnable runnable) {
+        if (runnable == null) {
+            throw new NullPointerException("Trying to register null runnable");
```

Comment:
```suggestion
            throw new NullPointerException("Cannot register null value as runnable for conditional configuration. Please ensure that all values you register are not null.");
```

Reply:
I updated all messages following this pattern.